src/risedevtool/src/task/meta_node_service.rs

```diff
@@ -16,16 +16,22 @@ use std::env;
 use std::path::{Path, PathBuf};
 use std::process::Command;
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, bail, Context, Result};
 use itertools::Itertools;
+use sqlx::{ConnectOptions, Database};
+use url::Url;
 
 use super::{risingwave_cmd, ExecuteContext, Task};
-use crate::util::{get_program_args, get_program_env_cmd, get_program_name};
+use crate::util::{get_program_args, get_program_env_cmd, get_program_name, is_env_set};
 use crate::{
     add_hummock_backend, add_tempo_endpoint, Application, HummockInMemoryStrategy, MetaBackend,
     MetaNodeConfig,
 };
 
+fn sql_endpoint_from_env() -> Result<String> {
```

Comment:
better to add commends about how's the value look like

Reply:
Added